webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -118,30 +129,44 @@ class SubscriptionsPage extends Component {
 
                   <div className="toolbar-pf-action-right">
                     <FormGroup>
-                      <LinkContainer to="subscriptions/add">
-                        <Button bsStyle="primary">
+                      <LinkContainer to="subscriptions/add" disabled={disableManifestActions}>
+                        <DisabledButtonWithTooltip
+                          id="add-subscriptions-button"
+                          disabled={disableManifestActions}
+                          disabledReason={disabledReason}
+                          bsStyle="primary"
```

Comment:
Is the `bsStyle` and `id` actually propagated to the button? The props doesn't seem to be passed down.

Reply:
Update: I see where the `id` goes now. But `bsStyle` is ignored.

Reply:
Good catch, will fix.